shell.nix

```diff
@@ -0,0 +1,46 @@
+let
+  here = toString ./.;
+  moz_overlay = import (builtins.fetchTarball
+    "https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz");
+  pkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
+  rust = (pkgs.rustChannelOf {
+    channel = "nightly";
+  }).rust.override {
+    extensions = [ "rust-src" "rust-analysis" ];
+    targets = [ "thumbv7em-none-eabihf" ];
+  };
+  rustPlatform = pkgs.makeRustPlatform {
+    rustc = rust;
+    cargo = rust;
+  };
+
+
+
+  my-python-packages = p: with p; [
+    pyserial
```

Comment:
As of #54 there should be much less of a need for Python, but there is still [`crates/build-examples.py`](https://github.com/flipperzero-rs/flipperzero/blob/48392243271f7b9151050ed5f79feca621a5f26b/crates/build-examples.py) helper.

I believe most Python installations are about 500 MB in size. Do we have to be wary about size at all?